<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::Entity::EntityFunctionTables::EditorWidget</class>
 <widget class="QMainWindow" name="UnTech::GuiQt::Entity::EntityFunctionTables::EditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity ROM Structs Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GuiQt::Accessor::ListAccessorTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QToolBar" name="tableButtons">
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="floatable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiQt::Accessor::ListAccessorTableView</class>
   <extends>QTreeView</extends>
   <header>gui-qt/accessor/listaccessortableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
